test/html/unitTests.html

```diff
@@ -25,17 +25,17 @@
     <script src="../../js/site.js"></script>
     <script src="../../js/utils.js"></script>
     <script src="../../js/indexed-db.js"></script>
+    <script src="../../js/url-parse.js"></script>
```

Comment:
Same comment as in other tldjs PR -- let's just wire tldjs by name and include it here as `js/tldjs`.